app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -50,6 +51,9 @@ const MakeDecision = ({ data }) => {
   const dispatch = useDispatch();
   const activeProject = useSelector(activeProjectSelector);
   const historyItems = useSelector(historyItemsSelector);
+  const isAnalyzerAvailable = useSelector(analyzerExtensionsSelector).find(
```

Comment:
Why not just check `analyzerExtensions` length?

Reply:
How can I be sure that an existing extension is a analyzer?